deps.edn

```diff
@@ -103,40 +101,40 @@
-  org.apache.logging.log4j/log4j-api        {:mvn/version "2.24.0"}             ; add compatibility with log4j 1.2
-  org.apache.logging.log4j/log4j-core       {:mvn/version "2.24.0"}             ; apache logging framework
-  org.apache.logging.log4j/log4j-jcl        {:mvn/version "2.24.0"}             ; allows the commons-logging API to work with log4j 2
-  org.apache.logging.log4j/log4j-jul        {:mvn/version "2.24.0"}             ; java.util.logging (JUL) -> Log4j2 adapter
+  org.apache.logging.log4j/log4j-1.2-api    {:mvn/version "2.24.2"}             ; apache logging framework
+  org.apache.logging.log4j/log4j-api        {:mvn/version "2.24.2"}             ; add compatibility with log4j 1.2
+  org.apache.logging.log4j/log4j-core       {:mvn/version "2.24.2"}             ; apache logging framework
+  org.apache.logging.log4j/log4j-jcl        {:mvn/version "2.24.2"}             ; allows the commons-logging API to work with log4j 2
+  org.apache.logging.log4j/log4j-jul        {:mvn/version "2.24.2"}             ; java.util.logging (JUL) -> Log4j2 adapter
   org.apache.logging.log4j/log4j-slf4j2-impl
-  {:mvn/version "2.24.0"}             ; allows the slf4j2 API to work with log4j 2
+  {:mvn/version "2.24.2"}             ; allows the slf4j2 API to work with log4j 2
   org.apache.logging.log4j/log4j-layout-template-json
-  {:mvn/version "2.24.0"}             ; allows the custom json logging format
+  {:mvn/version "2.24.2"}             ; allows the custom json logging format
   org.apache.poi/poi                        {:mvn/version "5.3.0"}              ; Work with Office documents (e.g. Excel spreadsheets) -- newer version than one specified by Docjure
   org.apache.poi/poi-ooxml                  {:mvn/version "5.3.0"
                                              :exclusions  [org.bouncycastle/bcpkix-jdk15on
                                                            org.bouncycastle/bcprov-jdk15on]}
-  org.apache.poi/poi-ooxml-full             {:mvn/version "5.2.5"}
-  org.apache.sshd/sshd-core                 {:mvn/version "2.13.2"              ; ssh tunneling and test server
+  org.apache.poi/poi-ooxml-full             {:mvn/version "5.3.0"}
+  org.apache.sshd/sshd-core                 {:mvn/version "2.14.0"              ; ssh tunneling and test server
                                              :exclusions  [org.slf4j/slf4j-api
                                                            org.slf4j/jcl-over-slf4j]}
-  org.apache.tika/tika-core                 {:mvn/version "2.9.2"}
-  org.apache.xmlgraphics/batik-all          {:mvn/version "1.17"}               ; SVG -> image
-  org.bouncycastle/bcpkix-jdk18on           {:mvn/version "1.78.1"}             ; Bouncy Castle crypto library -- explicit version of BC specified to resolve illegal reflective access errors
-  org.bouncycastle/bcprov-jdk18on           {:mvn/version "1.78.1"}
+  org.apache.tika/tika-core                 {:mvn/version "3.0.0"}
```

Comment:
This mentions a bunch of breaking changes, but I guess we're only using it to detect a very narrow range of formats so I guess we can trust our tests.